### PR TITLE
Add `vtk.js` to Atlascope

### PR DESCRIPTION
**Pull request by naglepuff**

### CODE REVIEW

File: atlascope/core/management/commands/populate.py [hunk 1]
```diff
@@ -109,6 +109,7 @@ def populate_investigations(specs):
         datasets = [Dataset.objects.get(name=name) for name in spec["datasets"]]
         # Only use datasets whose content saved properly
         datasets = [dataset for dataset in datasets if dataset.content]
+        # datasets = [dataset for dataset in datasets if dataset.content]
```
Comment (annehaley) [inline atlascope/core/management/commands/populate.py]: This addition can be left out

File: web/src/components/InvestigationSidebar.vue [hunk 2]
```diff
@@ -35,6 +35,19 @@
...
+      <v-tab-item>
+        <div class="ma-2 pa-2">
+          <router-link
+            :to=" { name: 'vtkViewer' }"
+            target="_blank"
+          >
+            Launch VTKjs
+          </router-link>
+        </div>
+      </v-tab-item>
```
Comment (annehaley) [inline web/src/components/InvestigationSidebar.vue:50]: Are we going to offer either vtk.js OR paraview glance to visualize 3d data?

File: web/src/views/InvestigationDetail.vue [hunk 3]
```diff
@@ -592,6 +592,13 @@ export default defineComponent({
       }
     }
 
+    function toggle3DVolumePin(dataset: Dataset) {
+      const apiRoot = process.env.VUE_APP_API_ROOT;
+      const datasetUrl = `${apiRoot}/datasets/${dataset.id}/download`;
+      const glanceUrl = `http://localhost:9999/?name=${dataset.name}&url=${datasetUrl}`;
+      window.open(glanceUrl, '_blank');
+    }
```
Comment (annehaley) [inline web/src/views/InvestigationDetail.vue]: I am a bit confused about when we open something in glance vs vtk.js

**Review by annehaley — CHANGES_REQUESTED** [review]

**Comment by naglepuff** [comment]
Closing in favor of #200. This could be cool to come back to at some point if we want to add multiple visualization options. At this point in the project, #200 does a good job of showing how we can integrate other tools or services to allow for interacting with Atlascope data.

**Pull request closed without merging**